web/frontend/src/components/gantry/index.svelte

```diff
@@ -1,133 +1,253 @@
 <script lang="ts">
 
-import { gantryApi } from '@viamrobotics/sdk';
-import { displayError } from '@/lib/error';
-import { rcLogConditionally } from '@/lib/log';
-import Collapse from '@/lib/components/collapse.svelte';
-import { useRobotClient } from '@/hooks/robot-client';
-
-export let name: string;
-export let status: {
-  is_moving: boolean
-  lengths_mm: number[]
-  positions_mm: number[]
-} = {
-  is_moving: false,
-  lengths_mm: [],
-  positions_mm: [],
-};
-
-const { robotClient } = useRobotClient();
```

Comment:
it looks like a lot of recent changes are reverted in this PR 😨 merge conflict resolution gone bad?

Reply:
yeah working through that right now

Reply:
got it, I'll wait!

Reply:
okay I think I fixed it all but I might have missed some things? let me know if you see any updates still reverted